cmd/snap-update-ns/mount-entry.h

```diff
@@ -30,6 +30,7 @@
 struct sc_mount_entry {
 	struct mntent entry;
 	struct sc_mount_entry *next;
+	unsigned flag;		// general-purpose flag, not compared
```

Comment:
If this is used for more in the future, maybe we can change this code here once this will actually be changed in the future? YAGNI and all :) ? Or do we know already that it will change and in what way?